src/bouwdossiers/serializers.py

```diff
@@ -53,18 +52,13 @@ def to_representation(self, instance):
         result = super().to_representation(instance)
         _bestanden = []
 
-        for bestand in result["bestanden"]:
+        for index, bestand in enumerate(result["bestanden"]):
```

Comment:
Hier ga je ervanuit dat de volgorde van de bestanden in `result` altijd hetzelfde is, klopt dat? Of maakt het niet uit?